_saas-integrations/workday/v1/workday-raas-v1.md

```diff
@@ -0,0 +1,133 @@
+status-url: ""
+
+tap-name: "workday-raas"
+repo-url: https://github.com/singer-io/tap-workday-raas
+
+this-version: "1"
+
+api: |
+  [SOAP API](https://community.workday.com/api){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true
+
+api-type: "platform.workday-raas" 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Enterprise"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. There are no schema files for this integration.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+setup-steps:
+  - title: "Retrieve your report URL."
+    anchor: "retrieve-report-url"
+    content: |
+      1. Login to your {{ integration.display_name }} account.
+      2. Navigate to your reports.
+      3. Select the report you want to replicate in Stitch and ensure that the **Enable As Web Service** box is checked.
+      4. Copy the report URL and keep it available for the next step.
+
+      {% include note.html type=single-line content="**Note**: If you would like to replicate multiple reports, repeat steps 3 & 4." %}
+
+  - title: "add integration"
+    content: |
+      4. In the **Username** field, enter the username for your {{ integration.display_name }} account.
+      5. In the **Password** field, enter the password for your {{ integration.display_ name }} account.
+      6. In the **Report** field, enter the name of your report.
+      7. In the **Report** field, enter the report URL you obtained in [step 1](#retrieve-report-url).
+
+      {% include note.html type=single-line content="**Note**: If you would like to replicate more reports, click on **Configure another report**." %}
+
+  - title: "historical sync"
+  - title: "replication frequency"
+  - title: "track data" ## remove this if the integration doesn't support at least table selection
+
+
+# -------------------------- #
+#     Integration Tables     #
+# -------------------------- #
+
+# Looking for the table schemas & info?
+# Each table has a its own .md file in /_integration-schemas/saas-integration
+
+
+# Remove this if you don't need it:
+# schema-sections:
+#  - title: ""
```

Comment:
What data should users expect to see when it's loaded into the destination? How are attributes identified by Stitch and replicated? Users want to know this information and outlining it here can help us shape their expectations.

It deals with the reports in Workday. Start there, and see if you can explain how Stitch will identify data points (attributes) in the report. This will also determine how the data looks in the destination.